ote/src/clj/ote/nap/users.sql

```diff
@@ -13,6 +13,21 @@
  LEFT JOIN "group" g ON g.id = m.group_id
      WHERE u.name = :username;
 
+-- name: fetch-user-by-email
```

Comment:
There are <number not printed here> of users with `null` email. This may therefore return a set of records, which is not intended.

Reply:
It is not possible to have null email and state='active'. Or maybe it is theoretically but in production there are non that are valid user.

Reply:
Thus select "where" should check that
state="active" AND email IS NOT NULL